docs/sdkx-xamarin/migration-guide.mdx

```diff
@@ -0,0 +1,527 @@
+---
+sidebar_position: 140
+title: Migration Guide
+description: "This migration guide will walk you through the steps you need to take in order to migrate from Helpshift legacy SDK to SDK X."
+---
+
+import {
+  Admonition,
+  CodeBlock,
+  Tabs,
+  TabItem,
+  Intro,
+  Steps,
+  Step,
+} from "@site/src/components/forDocs";
+
+# Migration Guide
+
+<Intro>
+
+SDK X Xamarin plugin is a Hybrid SDK that helps you roll out innovation faster by enabling most of the updates over the air that flow to end users without any downtime or app updates.
+
+This migration guide will walk you through the steps you need to take in order to migrate from Helpshift legacy Xamarin plugin to SDK X Xamarin plugin.
+
+</Intro>
+
+## Steps to migrate to SDK X {#steps-to-migrate}
+
+<Steps title="Migrate from legacy SDK to SDK X">
+  <Step
+    number="1"
+    title="Check migration support"
+    subtitle="Go through migration guide"
+    link="/sdkx-xamarin/migration-guide/#unsupported-features"
+    newTab={true}/>
+  <Step
+    number="2"
+    title="Update to SDK X"
+    subtitle="Migrate code/features step by step"
+    link="/sdkx-xamarin/migration-guide/#update-helpshift-dependencies"
+    newTab={true}/>
+  <Step
+    number="3"
+    title="Test the migration"
+    subtitle="Validate the functionality"/>
+</Steps>
+
+## Unsupported features {#unsupported-features}
+
+We recommend to hold off on SDK X migration if you are currently using the following APIs/features -
+
+- Guided issue filing ([iOS](/xamarin/support-tools-ios/#dynamic-forms)) ([Android](/xamarin/support-tools-android/#dynamic-forms)).
+- FAQ filtering by tags ([iOS](/xamarin/support-tools-ios/#faq-filtering)) ([Android](/xamarin/support-tools-android/#faq-filtering)).
+- Minimum Issue description length ([iOS](/xamarin/sdk-configuration-ios/#issue-description-minimum-length)) ([Android](localhost:3000/xamarin/sdk-configuration-android/#minimum-issue-desc-length)).
+- SDK UI strings customization ([iOS](/xamarin/string-customization-ios/)) ([Android](/xamarin/string-customization-android/)).
+- Automated review requests ([iOS](/xamarin/reviews-and-feedback-ios/#configuring-automatic-review-requests)) ([Android](/xamarin/reviews-and-feedback-android/#configure-review-request)).
+- Switching between light and dark themes ([iOS](/xamarin/design-ios/#skinning)) ([Android](/xamarin/design-android/#skinning)).
+
+These features are yet to be developed, and we may add them to our roadmap. You can reach out to support@helpshift.com to raise any query for the above.
```

Comment:
which line ?

Reply:
This (line #59) - 
> These features are yet to be developed, and we may add them to our roadmap. You can reach out to support@helpshift.com to raise any query for the above.

Because this information is already covered in the note on line #50 I had proposed.

Reply:
done